src/IronWorker.php

```diff
@@ -76,10 +76,11 @@ public function __construct($config = null)
      * @return bool Result of operation
      * @throws \Exception
      */
-    public function upload($directory, $run_filename, $code_name, $options = array())
+    public function upload($directory, $run_filename, $code_name, $options = array(), $ignored = [])
```

Comment:
@thousandsofthem 
I didn't include "ignored" into "options" because options are passed to backend in request body. Do we need to send "ignored file list" to the backend (as a part of options)?